pkg/collector/corechecks/snmp/topology_integration_test.go

```diff
@@ -0,0 +1,2097 @@
+		"1.3.6.1.2.1.2.2.1.13",
+		"1.3.6.1.2.1.2.2.1.14",
+		"1.3.6.1.2.1.2.2.1.2",
+		"1.3.6.1.2.1.2.2.1.6",
+		"1.3.6.1.2.1.2.2.1.7",
+		"1.3.6.1.2.1.2.2.1.8",
+		"1.3.6.1.2.1.31.1.1.1.1",
+		"1.3.6.1.2.1.31.1.1.1.18",
+		"1.3.6.1.2.1.4.20.1.2",
+		"1.3.6.1.2.1.4.20.1.3",
+		"1.3.6.1.4.1.9.9.23.1.2.1.1.17",
+		"1.3.6.1.4.1.9.9.23.1.2.1.1.19",
+		"1.3.6.1.4.1.9.9.23.1.2.1.1.20",
+		"1.3.6.1.4.1.9.9.23.1.2.1.1.5",
+		"1.3.6.1.4.1.9.9.23.1.2.1.1.6",
+		"1.3.6.1.4.1.9.9.23.1.2.1.1.7",
+	}, checkconfig.DefaultBulkMaxRepetitions).Return(&bulkPacket, nil)
+
+	err = chk.Run()
+	assert.Nil(t, err)
+
+	// language=json
+	event := []byte(fmt.Sprintf(`
+{
+  "subnet": "",
+  "namespace":"profile-metadata",
+  "devices": [
+    {
+      "id": "profile-metadata:1.2.3.4",
+      "id_tags": [
+        "device_namespace:profile-metadata",
+        "snmp_device:1.2.3.4"
+      ],
+      "tags": [
+        "agent_version:%s",
+        "device_namespace:profile-metadata",
+        "device_vendor:f5",
+        "snmp_device:1.2.3.4",
+        "snmp_host:foo_sys_name",
+        "snmp_profile:f5-big-ip"
+      ],
+      "ip_address": "1.2.3.4",
+      "status": 1,
+      "name": "foo_sys_name",
+      "description": "BIG-IP Virtual Edition : Linux 3.10.0-862.14.4.el7.ve.x86_64 : BIG-IP software release 15.0.1, build 0.0.11",
+      "sys_object_id": "1.2.3.4",
+      "location": "paris",
+      "profile": "f5-big-ip",
+      "vendor": "f5",
+      "serial_number": "a-serial-num",
+      "version":"15.0.1",
+      "product_name":"BIG-IP",
+      "model":"Final",
+      "os_name":"LINUX (3.10.0-862.14.4.el7.ve.x86_64)",
+      "os_version":"3.10.0-862.14.4.el7.ve.x86_64",
+      "os_hostname":"my-linux-f5-server"
+    }
+  ],
+  "interfaces": [
+    {
+      "device_id": "profile-metadata:1.2.3.4",
+      "id_tags": ["interface:nameRow1"],
+      "index": 1,
+      "name": "nameRow1",
+      "alias": "descRow1",
+      "description": "ifDesc1",
+      "mac_address": "82:a5:6e:a5:c9:01",
+      "admin_status": 1,
+      "oper_status": 1
+    },
+    {
+      "device_id": "profile-metadata:1.2.3.4",
+	  "id_tags": ["interface:nameRow2"],
+      "index": 2,
+      "name": "nameRow2",
+      "alias": "descRow2",
+      "description": "ifDesc2",
+      "mac_address": "82:a5:6e:a5:c9:02",
+      "admin_status": 1,
+      "oper_status": 1
+    }
+  ],
+  "ip_addresses": [
+    {
+      "interface_id": "profile-metadata:1.2.3.4:1",
+      "ip_address": "10.0.0.1",
+      "prefixlen": 24
+    },
+    {
+      "interface_id": "profile-metadata:1.2.3.4:1",
+      "ip_address": "10.0.0.2",
+      "prefixlen": 24
+    }
+  ],
+  "links": [
+        {
+            "id": "profile-metadata:1.2.3.4:1.5",
+            "source_type": "cdp",
+            "local": {
+                "device": {
+                    "dd_id": "profile-metadata:1.2.3.4"
+                },
+                "interface": {
+                    "dd_id": "profile-metadata:1.2.3.4:1",
+					"id": ""
```

Comment:
🥜 Some indent inconsistencies:
![image](https://user-images.githubusercontent.com/49917914/228829623-58f0f14c-face-4683-af05-ea3bca6ebe1c.png)

Reply:
I think it's the github diff view 🤔 
I have the correct indentation in my IDE and also when I [open the file ](https://github.com/DataDog/datadog-agent/blob/47229feff4850df8d6fc105832c5d3616fea1006/pkg/collector/corechecks/snmp/topology_integration_test.go#L1356-#L1359)